common/debug-interface/src/libra_trace.rs

```diff
@@ -296,3 +333,60 @@ fn abbreviate_crate(name: &str) -> &str {
         _ => name,
     }
 }
+
+// This is exact copy of similar function in log crate
+/// Sets libra trace config
+pub fn set_libra_trace(config: &HashMap<String, String>) -> Result<(), ()> {
+    unsafe {
```

Comment:
We should probably parse config outside of unsafe block(and return error, change this fn to anyhow::Result instead of Result<(), ()>).